extensions/quickgpt/package.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "quickgpt",
+  "title": "quickgpt",
+  "description": "quickgpt is a tool to provide prompt for chatgpt.",
+  "icon": "command-icon.png",
+  "author": "ddhjy2012",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "index",
+      "description": "运行quickgpt工具，该工具用于为 chatgpt 提供提示词。需要一个视图，并接受三个可选参数：剪贴板文本、选择文本和动作。",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English.